lib/src/intl_suggestors/intl_message_syntax.dart

```diff
@@ -139,9 +144,30 @@ class MessageSyntax {
     return '(${args.join(', ')})';
   }
 
+  // This will ignore the first few interpolated variable
+  /// Input: $numHiddenPills additional values not shown'
+  /// Output: static String additionalValuesNotShown(String numHiddenPills)
+  String ignoreInterpolatedVariable(String messageText) {
+    if (messageText.startsWith("'") && messageText.endsWith("'")) {
+      messageText = messageText.substring(1, messageText.length - 1);
+    }
+    if (messageText.startsWith('\$') && messageText.contains(' ')) {
+      String toRemoveFromName = messageText
+          .trim()
+          .substring(messageText.indexOf('\$'), messageText.indexOf(" "));
+      messageText = messageText.replaceFirst(toRemoveFromName, "").trim();
+    }
+    return messageText;
+  }
+
   String nameForNode(StringLiteral body,
       {String? initialName, bool startAtZero = false}) {
     var messageText = body.toSource();
+    var removingInterpolatedVariable = ignoreInterpolatedVariable(messageText);
+    if (removingInterpolatedVariable.isNotEmpty &&
+        removingInterpolatedVariable.contains(' ')) {
+      initialName = toVariableName(removingInterpolatedVariable);
```

Comment:
This is the reason the constant tests are failing. You're overwriting the parameter initialName, and in those tests it's being passed in.

Reply:
I updated the code based on your suggestion as [above](https://github.com/Workiva/over_react_codemod/pull/193#discussion_r1015730786) and now `constant tests` are running successfully.